sql/core/src/main/java/org/apache/spark/sql/vectorized/ArrowColumnVector.java

```diff
@@ -53,166 +41,83 @@ public int numNulls() {
   @Override
   public void close() {
     if (childColumns != null) {
-      for (int i = 0; i < childColumns.length; i++) {
-        childColumns[i].close();
+      for (ArrowColumnVector childColumn : childColumns) {
```

Comment:
Will this be faster?

Reply:
Should also apply similar change to `WritableColumnVector`

Reply:
the performance is same, it's just a more standard way to iterate an array in java